server/connector/key_utils.hpp

```diff
@@ -20,6 +20,7 @@
 
 #pragma once
 
+#include <absl/base/internal/endian.h>
```

Comment:
is this used? I guess we moved endian related logic into primary_key::AppendSigned